frontend/src/components/dialogs/SecretDialogGeneric.vue

```diff
@@ -0,0 +1,165 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+ <template>
+  <secret-dialog
+    :value=value
+    :data="secretData"
+    :data-valid="valid"
+    :secret="secret"
+    :vendor="vendor"
+    :create-title="`Add new ${vendor} Secret`"
+    :replace-title="`Replace ${vendor} Secret`"
```

Comment:
on replace, the secret is shown as
```
Secret Data
{}```
however on reload it shows the secret data

Reply:
probably temporary reactivity problem